posydon/binary_evol/CE/step_CEE.py

```diff
@@ -540,364 +947,476 @@ def CEE_simple_alpha_prescription(
             print("separation_i in Rsun", separation_i/const.Rsun)
             print("separation_postCEE in Rsun", separation_postCEE/const.Rsun)
 
-        # now we check if the roche Lobe of any of the cores that spiralled-in
-        # will be filled if reached this final separation
-        RL1 = cf.roche_lobe_radius(mc1_i, mc2_i, separation_postCEE/const.Rsun)
-        RL2 = cf.roche_lobe_radius(mc2_i, mc1_i, separation_postCEE/const.Rsun)
+
+        # Calculate the post-CE binary properties
+        if common_envelope_option_after_succ_CEE == \
+                "CEE_one_phase_variable_core_definition":
```

Comment:
Just for a consistent style in this file:
```suggestion
        if (common_envelope_option_after_succ_CEE
            == "CEE_one_phase_variable_core_definition"):
```